### PR TITLE
Framework: yml file to enable a gcc-openmpi container build that can run for PR

### PR DESCRIPTION
**Pull request by fryeguy52**
@trilinos/framework 

## Motivation
This will enable a containerized build through github actions. The idea is that it will be run for all PR

### CODE REVIEW

File: .github/workflows/PR-gcc-openmpi.yml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        run: |
+          bash -l -c "./get_dependencies.sh --container"
+      - name: Generate PR cmake fragment
+        working-directory: /home/Trilinos/build/
+        run: |
+          bash -l -c "source /home/Trilinos/src/Trilinos/packages/framework/GenConfig/gen-config.sh rhel8_gcc-openmpi_debug_shared_no-kokkos-arch_no-asan_complex_no-fpic_mpi_no-pt_no-rdc_no-uvm_deprecated-on_no-package-enables --cmake-fragment PR-configuration.cmake /home/Trilinos/src/Trilinos"
+      - name: Generate enable packages cmake fragment
+        working-directory: /home/Trilinos/build/
+        run: |
+          bash -l -c "TRILINOS_DIR=/home/Trilinos/srs/Trilinos /home/Trilinos/srs/Trilinos/commonTools/framework/get-changed-trilinos-packages.sh HEAD develop package-enables.cmake ctest-labels.cmake"
```
Comment (sebrowne) [inline .github/workflows/PR-gcc-openmpi.yml:57]: 'develop' will need to be dynamic, the merge target won't always be the develop branch.  This is fine for prototyping though.

**Review by sebrowne — APPROVED** [review]

**Review by sebrowne — COMMENTED** [review]
Why are so many tests failing?  Can you get output from a couple of them to see if we need to change something in the container?

**Review by sebrowne — DISMISSED** [review]
The failing tests are small enough now in magnitude that I'm okay with this.  Want to get those figured out pretty soon though.

File: packages/framework/ini-files/config-specs.ini [hunk 2]
```diff
@@ -2848,13 +2849,15 @@ use PACKAGE-ENABLES|NO-PACKAGE-ENABLES
 
 use COMMON_SPACK_TPLS
 
-opt-set-cmake-var MPI_EXEC_PRE_NUMPROCS_FLAGS                            STRING       : --bind-to;none
```
Comment (sebrowne) [inline packages/framework/ini-files/config-specs.ini:2851]: I'd like us to stop aligning columns like this for exactly this reason.  It generates a large diff when you really changed a couple of things.  Don't worry about it for this one, but for the future I do not think keeping the columns aligned is a good idea (https://stackoverflow.com/questions/101958/code-formatting-is-lining-up-similar-lines-ok agrees with me, though that's not exactly an iron-clad source).

**Review by sebrowne — APPROVED** [review]

**Comment by sebrowne** [comment]
@cgcgcg thank you for catching that!  I will reach out to Eric and get that changed to PreTest-Inspectors

**Comment by triple3567** [comment]
@cgcgcg should be fixed now. Anyone in the "pretest-inspectors" group can start builds by setting the "AT: PRE-TEST INSPECTED" Label, or by opening a new PR, or by committing changes to an existing PR.

**Comment by cgcgcg** [comment]
@triple3567 Is there a way to trigger this not only by setting "AT: PRE-TEST INSPECTED" but also by approving a PR? I tried this on #12491 and before the label was set the action failed early on. Is that the expected behavior?

**Pull request merged**